client/components/mma/cancel/CancellationJourneyFunnel.tsx

```diff
@@ -36,9 +33,20 @@ export const CancellationJourneyFunnel = () => {
 		return <Navigate to="/" />;
 	}
 
+	const productType = getSpecificProductTypeFromTier(productDetail.tier);
+	const productTypeKey = productType.productType;
+
+	const possiblePaidPlan = productDetail.subscription.currentPlans[0];
+	const qmEventString = `cancellation start | ${productType.friendlyName}${
+		isPaidSubscriptionPlan(possiblePaidPlan)
+			? ` | billing period: ${possiblePaidPlan.billingPeriod}`
+			: ''
+	}`;
+	window.QuantumMetricAPI?.sendEvent(184, 0, qmEventString);
```

Comment:
for clarity prefer to keep the structure until we need to use it?
might need to tweak my example to make it more typescripty though
```suggestion
	const qmEvent = [
	  `cancellation start`,
	  productType.friendlyName,
	];
	if (isPaidSubscriptionPlan(possiblePaidPlan)
			qmEvent.push(`billing period: ${possiblePaidPlan.billingPeriod}`);
	}
	window.QuantumMetricAPI?.sendEvent(184, 0, qmEventString.join(" | ");
```

Reply:
yep, I've refactored this in to something similar